onnxruntime_extensions/_hf_cvt.py

```diff
@@ -287,7 +289,43 @@ def pre_processing(self, **kwargs):
         return g
 
     def post_processing(self, **kwargs):
+        with_default_inputs = kwargs.pop("WITH_DEFAULT_INPUTS", True)
+
         _cvt_op = self.cvt_quadruple.post_op
         _cvt_func = self.cvt_quadruple.post_attribute_cvt
         cvt = partial(_cvt_func, self.cvt_obj)
-        return SingleOpGraph.build_graph(_cvt_op, cvt=cvt, **kwargs)
+        g = SingleOpGraph.build_graph(_cvt_op, cvt=cvt, **kwargs)
+
+        default_inputs = {}
+        if with_default_inputs:
+            op_class = SingleOpGraph.get_op_class(_cvt_op)
```

Comment:
Can you add a new decoder operator which contains the tokenizer.json and tokenizer_config.json as what we did for encoder?

Reply:
yep, will do this and create a custom op like `HfJsonDetokenizer` in a different PR (as it will include C API changes as well).